controllers/incidents.js

```diff
@@ -5,52 +5,82 @@ exports.getIncidents = function(req, res) {
   allIncidents(req, res)
 };
 
-
 var allIncidents = function(req, res) {
-  var searchQuery = mapQuery(req.query);
+  var page = req.query["page"] || 1;
+  var offset = 100 * (page - 1);
+  var searchQuery = transformQuery(req.query);
+
   if (searchQuery["invalid_parameter"]) {
     return res.json({invalid_request: "unrecognized search parameter"})
   } else {
-    models.Incident.findAll({where: searchQuery, limit: 100, include: [ models.Beat, models.Disposition, models.CallType ]}).then(function(incidents) {
-      for(i in incidents) {
-        incidents[i] = incidents[i].serialize();
-      }
-      return res.json(incidents);
+    models.Incident.findAndCountAll({where: searchQuery,
+                                     offset: offset,
+                                     limit: 100,
+                                     order: '"date" ASC',
+                                     include: [ models.Beat, models.Disposition, models.CallType ]})
+                    .then(function(results) {
+                                    customizeJsonKeys(results);
+                                    serializeIncidents(results);
+                                    return res.json(results);
     });
   };
 }
 
-var queryHashMap = {number: "number",
-                    priority: "priority",
-                    beat: "Beat.number",
-                    neighborhood: "Beat.neighborhood",
-                    disp_code: "Disposition.code",
-                    disp_desc: "Disposition.description",
-                    call_code: "CallType.code",
-                    call_desc: "CallType.description",
-                    street: "street",
-                    street_number: "street_number",
-                    street_dir: "street_dir",
-                    date: "date",
-                    start_date: "date",
-                    end_date: "date"};
+var queryHashMap = {number:         "number",
```

Comment:
Borderline nitpick but you might be able to get more specific with this name, `queryHashMap` describes *what* it represents but doesn't really get at its intent or why it exists. I don't have any great suggestions off the top though :|.